composer/trainer/devices/device_tpu.py

```diff
@@ -27,9 +28,8 @@ class DeviceTPU(Device):
     """
 
     def __init__(self):
-        import torch_xla.core.xla_model as xm
-
-        self._device = xm.xla_device()
+        # Can't use xm.xla_device() api here!
+        self._device = torch.device(f'xla:0')
```

Comment:
Why can't we use it here?

Reply:
Updated comment. Let me know if it's still not clear.